### ISSUE TITLE
Last line of Doc covered by status bar on Firefox

### ISSUE THREAD

**Issue by mhdrone** [hedgedoc/hedgedoc#724]
I looked over the other issues and couldn't find one for this.

### Description

If the document is longer than the window (when the scroll bar appears), then the last line becomes covered by the status bar when using firefox.

Here is a screenshot of it on the demo server:
![last line covered by status bar on demo server](https://i.imgur.com/RVNbarL.png)

When accessing the same note on chrome the status bar does not become covered

### To Reproduce
1. Using firefox visit the Demo server
2. Add newlines until the scroll bar appears
3. If the currently selected line is still not covered by the status bar then select a line above the last one and re-select the last line.

### Expected behavior

The final line should be visible and not covered by the status line.

### Setup:
 - Host OS: Ubuntu 20.04
 - NodeJS version 10.19.0
 - HedgeDoc version
     - server 1.7.1

Issue is on the [demo server](https://demo.hedgedoc.org/) as well.

### Potential Fix
I have found a fix but I am not sure if it completely addresses the problem due to insufficient testing.

Edited `public/css/index.css`

Changed (starts on line 66)
```css
.CodeMirror-code {
   /*padding-bottom: 36px;*/
}
```

To
```css
.CodeMirror-code {
    margin-bottom: 36px;
}
```

Let me know what other info you might need.

**Comment by davidmehren** [comment]
Thanks for your report!
A fix was included in HedgeDoc 1.7.2.

**Comment by Genygo** [comment]
Hi,
when i write a new line at the bottom the new line is still covered. You could scroll down manually to see the full line but it didn't jump one full line down.
